x/crosschain/keeper/cctx_orchestrator_validate_outbound.go

```diff
@@ -325,21 +325,20 @@ func (k Keeper) processFailedZETAOutboundOnZEVM(ctx sdk.Context, cctx *types.Cro
 // TODO: consolidate logic with above function
```

Comment:
Can we adjust the comments for `processFailedOutboundV2` function to refect the changes

Reply:
[340bc98](https://github.com/zeta-chain/node/pull/3166/commits/340bc98ea3aae2c9b6893106f845daee0627a9eb)